atomic_reactor/plugins/post_pulp_pull.py

```diff
@@ -64,10 +64,18 @@ def run(self):
         pullspec = image.copy()
         pullspec.registry = registry.uri  # the image on Crane
 
+        media_types = []
+        for plugin in self.workflow.postbuild_plugins_conf:
+            if 'pulp_sync' in plugin['name']:
```

Comment:
Nitpick, but shouldn't this be an equality check? It can also leverage the constants: `PLUGIN_PULP_PUSH_KEY, PLUGIN_PULP_SYNC_KEY`